packages/client/hmi-client/src/services/model-configurations.ts

```diff
@@ -283,3 +283,9 @@ export function getModelInitials(modelConfiguration, source, amrInitials) {
 		};
 	});
 }
+
+// Get the model configuration name for the given model configuration id
+export function getModelConfigName(modelConfigs: ModelConfiguration[], configId: string) {
+	const modelConfig = modelConfigs.find((d) => d.id === configId);
+	return modelConfig?.name ?? '';
+}
```

Comment:
We should have a chat about how things are wired together, with the composables and utilities and such. For example you are calling this function inside a loop to get translation names, it might be better if we can cache `{ modelConfigId => name }` somewhere, then the process wouldn't need to be `O(n^2)`